scala/private/rule_impls.bzl

```diff
@@ -223,33 +219,31 @@ StatsfileOutput: {statsfile_output}
       ijar_out = ijar_output_path,
       ijar_cmd_path = ijar_cmd_path,
       srcjars = _join_path(all_srcjars.to_list()),
-      java_files = _join_path(java_srcs),
```

Comment:
Do we not need the java files in here?

Reply:
We don’t. Since we removed the java compiler shell-out, we don’t need a separate key. We do have them in the list of all sources.

Reply:
Which scalac needs to compile mixed java/scala targets.